src/subdomains/generic/user/models/user-data/user-data.service.ts

```diff
@@ -394,7 +403,7 @@ export class UserDataService {
     if (dto.nationality || dto.identDocumentId) {
       const existing = await this.userDataRepo.findOneBy({
         nationality: { id: dto.nationality?.id ?? userData.nationality?.id },
-        identDocumentId: dto.identDocumentId ?? userData.identDocumentId,
+        identDocumentId: dto.identDocumentId ?? userData.identDocumentId ?? '',
```

Comment:
I think we can't do that. We would find an existing every time we only update nationality without identDocumentId. I would remove the `?? ''` and would check in the if in 408 if there is an identDocumentId and if not, don't throw the ConflictException